pkg/proxy/proxier.go

```diff
@@ -319,38 +319,43 @@ type Proxier struct {
 	hostIP        net.IP
 }
 
+var (
+	ErrProxyOnLocalhost = fmt.Errorf("cannot proxy on localhost")
```

Comment:
it's OK.  I get it, I just wanted to probe you for what distinguishes.
This is LGTM

On Fri, May 15, 2015 at 9:29 AM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/proxy/proxier.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8269#discussion_r30423449>
> :
>
> > @@ -319,38 +319,43 @@ type Proxier struct {
> >  	hostIP        net.IP
> >  }
> >
> > +var (
> > +	ErrProxyOnLocalhost = fmt.Errorf("cannot proxy on localhost")
>
> Will add comments to the intended use (every consumer of NewProxier having
> the same validation error).
> … <#14d5868689f8dd5c_>
>  On May 15, 2015, at 12:05 PM, Tim Hockin <notifications@github.com>
> wrote: In pkg/proxy/proxier.go: > @@ -319,38 +319,43 @@ type Proxier struct
> { > hostIP net.IP > } > > +var ( > + ErrProxyOnLocalhost =
> fmt.Errorf("cannot proxy on localhost") Typed errors are fine (though it's
> not my favorite aspect of Go, whatevs). My point was to grok why some
> errors matter more than others. … — Reply to this email directly or view it
> on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8269/files#r30423449>
> .
>